build/swivel/Interfaces.sol

```diff
@@ -14,12 +14,16 @@ interface IErc4626 {
   function withdraw(uint256, address, address) external returns (uint256);
 }
 
-// TODO if we move away from adapters the `..Token` suffix can be dropped
-interface ICompoundToken {
```

Comment:
there's no reason to be overly specific with the interface names in Swivel.sol. These interfaces will be fine simply being named for what is in the Protocol Enum